cmd/operator-sdk/up/local.go

```diff
@@ -56,7 +56,7 @@ kubernetes cluster using a kubeconfig file.
 	upLocalCmd.Flags().StringVar(&operatorFlags, "operator-flags", "", "The flags that the operator needs. Example: \"--flag1 value1 --flag2=value2\"")
 	upLocalCmd.Flags().StringVar(&namespace, "namespace", "", "The namespace where the operator watches for changes.")
 	upLocalCmd.Flags().StringVar(&ldFlags, "go-ldflags", "", "Set Go linker options")
-	upLocalCmd.Flags().BoolVar(&debugFlag, "debug", false, "Start in debug mode")
+	upLocalCmd.Flags().BoolVar(&debugFlag, "enable-delve", false, "Start the operator using the delve debugger")
```

Comment:
One more nit: Can you also change the variable name from `debugFlag` to `enableDelve`?

Reply:
done